src/Calculator/Views/DateCalculator.xaml

```diff
@@ -502,7 +490,6 @@
                                 Grid.Row="2"
```

Comment:
It looks like there is one more TextBlock we can consolidate/remove like the others here.